Readme.md

```diff
@@ -36,7 +36,11 @@ TX and RX pin definitions are target-specific.  To function correctly, the code
         "rx": "PTD2"
  }
  ```
-3. Finally, locate the pins (i.e. PTC17, PTD2) on the board's [pinout picture](https://www.mbed.com/en/development/hardware/boards/nxp/frdm_k64f/#overview).
+
+3. Finally, locate the pins (i.e. PTC17, PTD2) on the board's pinout picture: ![pinout picture](https://developer.mbed.org/media/uploads/sam_grove/frdm-k64f-pinnout-1.jpg).
```

Comment:
1. limit the size of the picture so it does not take up too much of the screen. I would say about 700px. `![pinout picture](https://developer.mbed.org/media/uploads/sam_grove/frdm-k64f-pinnout-1.jpg =700x)`
1. Remove the dangling full stop at the end of the picture.